paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -21,11 +21,9 @@ data class Environment(
   val platformDir: String,
   val appTestDir: String,
   val resDir: String,
+  val assetsDir: String,
   val packageName: String,
-  val compileSdkVersion: Int
-) {
-  val assetsDir = "$appTestDir/src/main/assets/"
-}
+  val compileSdkVersion: Int)
 
```

Comment:
could the assets ever be in a location other than `$appTestDir/src/main/assets`?

Reply:
yes: either you are using another variant, or somehow decided to change the source-set folder in your gradle setup.
Also, `/src/main/assets` is not a _merged_ assets folder (as-in, inherites assets from dependencies).